[WebPubSub] Update readme to use consistent service description and introduction across all languages

vicancy opened the pull request:
# All SDK Contribution checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] **Please open PR in `Draft` mode if it is:**
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md).**

### [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code. (Track 2 only)
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK. Please double check nuget.org current release version.

## Additional management plane SDK specific contribution checklist: 
Note: Only applies to `Microsoft.Azure.Management.[RP]` or `Azure.ResourceManager.[RP]`
 
- [ ] Include updated [management metadata](https://github.com/Azure/azure-sdk-for-net/tree/main/eng/mgmt/mgmtmetadata).
- [ ] Update AzureRP.props to add/remove version info to maintain up to date API versions.

### Management plane SDK Troubleshooting
- If this is very first SDK for a services and you are adding new service folders directly under /SDK, please add `new service` label and/or contact assigned reviewer.
- If the check fails at the `Verify Code Generation` step, please ensure:
	- Do not modify any code in generated folders.
	- Do not selectively include/remove generated files in the PR.
	- Do use `generate.ps1/cmd` to generate this PR instead of calling `autorest` directly.
	Please pay attention to the @microsoft.csharp version output after running `generate.ps1`. If it is lower than current released version (2.3.82), please run it again as it should pull down the latest version.
	
	**Note: We have recently updated the PSH module called by `generate.ps1` to emit additional data. This would help reduce/eliminate the Code Verification check error. Please run following command**:

	    `dotnet msbuild eng/mgmt.proj /t:Util /p:UtilityName=InstallPsModules`

### Old outstanding PR cleanup
 Please note:
	If PRs (including draft) has been out for more than 60 days and there are no responses from our query or followups, they will be closed to maintain a concise list for our reviewers.

vicancy committed:
update readme to be consistent with service introduction

vicancy committed:
Avoid using client as it is a preserved term

Mohit-Chakraborty commented on sdk/webpubsub/Azure.Messaging.WebPubSub/README.md:
```diff
@@ -47,23 +43,23 @@ var serviceClient = new WebPubSubServiceClient(new Uri(endpoint), "some_hub", ne
 ### Connection
 
-Connections, represented by a connection id, represent an individual websocket connection to the Web PubSub service. Connection id is always unique.
+A connection, also known as a client or a client connection, represents an individual WebSocket connection connected to the Web PubSub service. When successfully connected, a unique connection ID is assigned to this connection by the Web PubSub service.
```
If I understand this right, you are saying that a connection can also be known as either 'client' or 'client connection'. Is that right?
I am not sure of the absolute equivalence between a 'connection' and a 'client'. Can you interchange the two words in all contexts for this service?
Also, could you add quotes around 'client connection' to clarify that the 2-word phrase is the replacement.

vicancy replied:
under the Web PubSub service context, "client" only stands for the "WebSocket client", so I try to use "serviceClient" when using this libarary to avoid confusion

vicancy replied:
Add a diagram trying to illustrate the case. Noticed that CI has a hard requirement about the title to use "Authenticate the [Cc]lient", can we suppress that?

vicancy committed:
Add a diagram to illustrate the architecture

Mohit-Chakraborty committed:
Add WebPubSub README to exclude list

'Client' has a special meaning to the service, so we don't want to use the section header 'Authenticate the client'

Mohit-Chakraborty commented on sdk/webpubsub/Azure.Messaging.WebPubSub/README.md:
```diff
@@ -33,11 +36,9 @@ dotnet add package Azure.Messaging.WebPubSub --prerelease
 - An existing Azure Web PubSub service instance.
 
-### Authenticate the client
+### Create and authenticate a WebPubSubServiceClient
```
```suggestion
### Create and authenticate a `WebPubSubServiceClient`
```

Mohit-Chakraborty commented on sdk/webpubsub/Azure.Messaging.WebPubSub/README.md:
```diff
@@ -47,23 +48,23 @@ var serviceClient = new WebPubSubServiceClient(new Uri(endpoint), "some_hub", ne
 ### Message
 
-Using this library, you can send messages to the client connections. A message can either be string text, JSON or binary payload.
+When the client is connected, it can send messages to the upstream application, or receive messages from the upstream application, through the WebSocket connection.
```
```suggestion
When a client is connected, it can send messages to the upstream application, or receive messages from the upstream application, through the WebSocket connection.
```

vicancy committed:
Update sdk/webpubsub/Azure.Messaging.WebPubSub/README.md

Co-authored-by: Mohit Chakraborty <8271806+Mohit-Chakraborty@users.noreply.github.com>

vicancy committed:
Update sdk/webpubsub/Azure.Messaging.WebPubSub/README.md

Co-authored-by: Mohit Chakraborty <8271806+Mohit-Chakraborty@users.noreply.github.com>
